CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,248 @@
+
+Proposals and improvements must be completed for all weeks unless a week's content specifies otherwise in the accompanying assignment `README`. 
+
+#### `A-` grades
+
+Citizens who do not meet the above requirements for two (2) weeks will be awarded a mark of `A-` if all other weeks contain a proposal and subsequent improvement(s).
+
+### The `B` tier
+
+(To qualify for this tier, all basic assignment objectives must be completely met.)
+
+Citizens who take the opportunity to both provide substantive improvements to assignment content qualify for this contract tier. In order to qualify for this tier, citizens must:
+
+* propose an improvement in each week's `proposal.md` file which:
+  * completes all document prompts
+  * must be finished by the start of a lab session (Friday) of a given week
+  * must be recognized _in writing_ by the Mayor by the end of the lab session
+  * must either incorporate or provide rationale for _not_ incorporating suggested revisions
+    * this work is due by the assignment deadline (Sunday of each week)
+
+These proposals may be revisions to existing content or creation of wholly new content. Proposals must be completed for all weeks unless a week's content specifies otherwise in the accompanying assignment `README`. 
+
+#### `B+` grades
+
+Citizens who both _propose and implement_ at least two (2) ideas will be awarded a mark of `B+` if all other weeks contain a proposal.
+
+#### `B-` grades
+
+Citizens who miss (2) proposals will be awarded a mark of `B-` if all other weeks contain them.
+
+### The `C` Tier
+
+(To qualify for this tier, all basic assignment objectives must be completely met.)
+
+As per the [the Basic Grade rule](#the-basic-grade), meeting 100% of a given assignment's objectives constitutes earning a `C` mark.
+
+#### `C+` grades
+
+Citizens who _propose_ at least (2) ideas will be awarded a mark of `C+`, given that all grading checks are met for all other weeks.
+
+### Grades lower than the Basic Grade
+
+There are three (3) instances that penalize evaluation:
+
+1. Incomplete assignments (i.e. basic expectations are not fulfilled)
+2. Ignored assignments (i.e. not completed at all)
+3. Plagiarism or other inappropriate/dishonest academic behavior
+
+Each of these circumstances affects grades differently.
+
+#### Incomplete assignments
+
+For every two (2) incomplete assignments, a citizen's course grade will be lowered by half a letter grade (e.g. _two_ (2) incomplete assignments would result in a mark of `C-`, et al.).
+
+#### Ignored assignments
+
+If a citizen misses _any_ assignment, their course grade is lowered by a half letter grade for each (e.g. _one_ (1) assignment missed would result in a `C-`, et al.).
```

Comment:
What if there is a valid excuse for a missed assignment?

Reply:
Communication to the Mayor should be done appropriately.

Reply:
Be that as it may, I feel some language that addresses that this isn't an absolute and that exceptions will be made by the Mayor on a case to case basis